TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/PruferTreeGeneratorTest.java

```diff
@@ -19,7 +19,10 @@ public void testThatArbitrarySequencePrintsExpectedTree() throws IOException {
         final int numNodes = 5;
         var edges = IntStream.range(0, numNodes-2).map(x->random.nextInt(numNodes)+1).toArray();
         var tree = ptg.makeTree(vn -> Integer.toString(vn), edges);
-        try (var baos = new ByteArrayOutputStream(); var printStream = new PrintStream(baos)) {
+        try (
+                var baos = new ByteArrayOutputStream();
+                var printStream = new PrintStream(baos, false, StandardCharsets.UTF_8)
+        ) {
```

Comment:
nit: I'm not sure why this was previously on 1 line & why it's now on 4.  Seems like it should be 2.  
Anyway - thanks for fixing this.  I wish that these weren't optional!